core/transaction_pool/impl/transaction_pool_impl.hpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_TRANSACTION_POOL_IMPL_HPP
+#define KAGOME_TRANSACTION_POOL_IMPL_HPP
+
+#include <outcome/outcome.hpp>
+#include "common/logger.hpp"
+#include "transaction_pool/pool_moderator.hpp"
+#include "transaction_pool/transaction_pool.hpp"
+
+namespace kagome::transaction_pool {
+
+  struct ReadyTransaction : public primitives::Transaction {
+    std::list<common::Hash256> unlocks;
+  };
+
+  struct WaitingTransaction : public primitives::Transaction {};
+
+  class TransactionPoolImpl : public TransactionPool {
+    static constexpr auto kDefaultLoggerTag = "Transaction Pool: ";
```

Comment:
make it TransactionPool. otherwise logline will look like 
```
[Transaction Pool: ] ...
```

Reply:
WIll it? I just tested it, it prints something like
[2019-07-01 13:18:57.309627653][th:31084][info] Transaction Pool:  Hello

Reply:
we potentially can change logger output format